portal/account/register.php

```diff
@@ -0,0 +1,556 @@
+<?php
```

Comment:
Needs license/copyright/author header. See here for an example of the new sleek method we are using:
http://www.open-emr.org/wiki/index.php/How_to_Document_Your_Code_Properly#File-Level_DocBlock